app/migrations/versions/9c233f12de2c_.py

```diff
@@ -29,6 +29,6 @@ def downgrade():
     # ### commands auto generated by Alembic - please adjust! ###
     op.add_column('transfer_account', sa.Column('owning_organisation_id', sa.INTEGER(), autoincrement=False, nullable=True))
     op.create_foreign_key('transfer_account_owning_organisation_id_fkey', 'transfer_account', 'organisation', ['owning_organisation_id'], ['id'])
-    op.drop_constraint(None, 'organisation', type_='foreignkey')
+    op.drop_constraint("fk_org_level_account", 'organisation', type_='foreignkey')
```

Comment:
Not ideal, but had to go back and modify a migration file because it kept on dying during the docker-compose launch

Reply:
how will that work for deploying to prod? can't rollback and re-migrate since you'd lose data right?

Reply:
also how do these work for migrating locally? do i have to drop my db and recreate?